packages/data-polling/src/useGetOperations.tsx

```diff
@@ -47,12 +48,14 @@ type QueryParams =
  *   hasMore - true if there are more operations to fetch
  *   loadMore - function to load more operations (older ones)
  */
-export const useGetOperations = (accounts: Account[]) => {
+export const useGetOperations = (accounts: (Account | ImplicitAccount)[]) => {
   const network = useSelectedNetwork();
   const dispatch = useAppDispatch();
   const refetchTrigger = useRefetchTrigger();
   const handleError = useReactQueryErrorHandler();
 
+  const isVerified = accounts.every(acc => acc.type === "mnemonic" && acc.isVerified);
```

Comment:
this breaks the behaviour for non-mnemonic account

I think we shouldn't update this piece at all. if you don't need the operations, just don't use the hook on the page

Reply:
we can't use useGetOperations conditionally